<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoDialog</class>
 <widget class="QDialog" name="infoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="Vse">
     <item>
      <layout class="QHBoxLayout" name="PAinnedetails">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Book Title:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="BooktitleGraph">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Author:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="AuthorGraph">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="PainPricRatingDescr">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Rating:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="RatingGraph">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Price:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="PriceGraph">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="TableDescriptionGenres"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="CartDetails">
       <item>
        <widget class="QTableWidget" name="TableDetails"/>
       </item>
       <item>
        <widget class="QPushButton" name="CartButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>add to cart</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
